secretstores/azure/keyvault/keyvault.go

```diff
@@ -25,9 +25,9 @@ const (
 )
 
 type keyvaultSecretStore struct {
-	vaultName        string
-	vaultClient      kv.BaseClient
-	clientAuthorizer ClientAuthorizer
+	vaultName   string
+	vaultClient kv.BaseClient
+	metadata    secretstores.Metadata
```

Comment:
With the other related review comments, then you no need to make metadata part of type.

Reply:
Solved in commit 8225aae.

Thanks for the review, this is a cleaner solution.